pkg/apis/pipeline/v1beta1/taskref_conversion.go

```diff
@@ -10,18 +10,51 @@ func (tr TaskRef) convertTo(ctx context.Context, sink *v1.TaskRef) {
 	sink.Name = tr.Name
 	sink.Kind = v1.TaskKind(tr.Kind)
 	sink.APIVersion = tr.APIVersion
-	// TODO: handle bundle in #4546
 	new := v1.ResolverRef{}
 	tr.ResolverRef.convertTo(ctx, &new)
 	sink.ResolverRef = new
+	tr.convertBundleToResolver(sink)
```

Comment:
May I ask if it's correct to put `convertBundletoResolver` after conversion for `ResolverRef` as there should be only 1 ResolverRef?

Reply:
we should have validation preventing a user from creating a taskref with both bundle and resolver, but if you'd like you can use conditionals here to ensure we don't convert both